src/data/integration.go

```diff
@@ -26,9 +26,7 @@ func CreateEntity(rabbitmqIntegration *integration.Integration, entityName, enti
 		name = joinVhostName(vhost, name)
 	}
 	metricNamespace = []attribute.Attribute{
-		{Key: "displayName", Value: name},
-		{Key: "entityName", Value: fmt.Sprintf("%s:%s", entityType, name)},
```

Comment:
why also getting rid of those? Not needed anymore?

Reply:
Since once of the issues is that `name` collisions between clusters deployed in k8s, i think it will make names less confusing if we leave [the Agent to decorate this values ](https://github.com/newrelic/infrastructure-agent/blob/master/pkg/integrations/legacy/runner.go#L745). But is this might add another breaking change which it might be not required to do.

Reply:
yeah, no idea if the best is to break "all at once" or avoid a further breaking change keeping a confusing metric there 😕

Reply:
As we talked, I'd suggest to not touch these for the time being. I do think it's better for the agent to generate this, but I don't see it's critical enough to risk breaking more things on this particular codebase.